From 366368ea2faa90899a7c94c8d8654b94633b14f6 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 5 Feb 2019 09:41:35 +0100
Subject: [PATCH] Squash of
 github/lineage-16.0(fa1692d9)..lineage-16.0-android-9.0.0_r31(29414e03) (DO
 NOT SUBMIT)

Bill Yi (5):
 - Import translations. DO NOT MERGE
 - Import translations. DO NOT MERGE
 - Import translations. DO NOT MERGE
 - Import translations. DO NOT MERGE
 - Import translations. DO NOT MERGE

Luca Stefani (1):
 - Merge tag 'android-9.0.0_r31' into lineage-16.0-android-9.0.0_r31

TreeHugger Robot (2):
 - Merge "Import translations. DO NOT MERGE" into pi-dev
 - Merge "Import translations. DO NOT MERGE" into pi-dev

android-build-team Robot (2):
 - Snap for 5073521 from b4760cd554004f6053b11edb4afefea9f5a99b53 to
   pi-qpr2-release
 - Snap for 5115288 from 1b573525a6db296081e52e536f62830e6e02f2c7 to
   pi-qpr2-release

Change-Id: I309899af046d102b3fc77fb7cb97228fb1542577

Change-Id: Ifcca01a30aa6e2ab960c8c60d488b854b20e793a
---
 res/values-as/strings.xml           | 2 +-
 res/values-bs/strings.xml           | 2 +-
 res/values-eu/inspector_strings.xml | 2 +-
 res/values-eu/mimes.xml             | 2 +-
 res/values-uz/inspector_strings.xml | 2 +-
 res/values-uz/strings.xml           | 2 +-
 6 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/res/values-as/strings.xml b/res/values-as/strings.xml
index 1d21c9a0..60537124 100644
--- a/res/values-as/strings.xml
+++ b/res/values-as/strings.xml
@@ -16,7 +16,7 @@
 
 <resources xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="files_label" msgid="771781190045103748">"ফাইলবোৰ"</string>
+    <string name="files_label" msgid="771781190045103748">"ফাইল"</string>
     <string name="downloads_label" msgid="5462789470049501103">"ডাউনল\'ডসমূহ"</string>
     <!-- no translation found for app_label (8089292432455111409) -->
     <skip />
diff --git a/res/values-bs/strings.xml b/res/values-bs/strings.xml
index 8d924a40..8625c7e4 100644
--- a/res/values-bs/strings.xml
+++ b/res/values-bs/strings.xml
@@ -84,7 +84,7 @@
     <string name="toast_no_application" msgid="7555319548595113121">"Nije moguće otvoriti fajl"</string>
     <string name="toast_view_in_archives_unsupported" msgid="1923221390170964845">"Nije moguće otvoriti fajlove u arhivama"</string>
     <string name="toast_failed_delete" msgid="3453846588205817591">"Nije moguće izbrisati neke dokumente"</string>
-    <string name="share_via" msgid="8725082736005677161">"Dijeli koristeći aplikaciju"</string>
+    <string name="share_via" msgid="8725082736005677161">"Dijeli pomoću aplikacije"</string>
     <string name="copy_notification_title" msgid="52256435625098456">"Kopiraju se fajlovi"</string>
     <string name="compress_notification_title" msgid="6830195148113751021">"Kompresovanje fajlova"</string>
     <string name="extract_notification_title" msgid="5067393961754430469">"Izdvajanje fajlova"</string>
diff --git a/res/values-eu/inspector_strings.xml b/res/values-eu/inspector_strings.xml
index 24b9b4a7..9f51a8c6 100644
--- a/res/values-eu/inspector_strings.xml
+++ b/res/values-eu/inspector_strings.xml
@@ -18,7 +18,7 @@
     xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
     <string name="inspector_title" msgid="1924760928091740238">"Informazioa"</string>
     <string name="inspector_load_error" msgid="7522190243413249291">"Ezin izan da kargatu fitxategiaren informazioa"</string>
-    <string name="inspector_debug_section" msgid="2576052661505700421">"Arazte-informazioa (garatzaileentzat soilik)"</string>
+    <string name="inspector_debug_section" msgid="2576052661505700421">"Arazketa-informazioa (garatzaileentzat soilik)"</string>
     <string name="inspector_debug_metadata_section" msgid="5875140675600744846">"Prozesatu gabeko metadatuak: <xliff:g id="METADATATYPE">%1$s</xliff:g>"</string>
     <string name="inspector_metadata_section" msgid="6077622515328240575">"Multimedia-elementuaren xehetasunak"</string>
     <string name="handler_app_file_opens_with" msgid="5272329600389613550">"Mota honetako fitxategiak aplikazio honekin ireki daitezke:"</string>
diff --git a/res/values-eu/mimes.xml b/res/values-eu/mimes.xml
index 45294b73..d5768f29 100644
--- a/res/values-eu/mimes.xml
+++ b/res/values-eu/mimes.xml
@@ -40,6 +40,6 @@
     <string name="gtable_file_type" msgid="7332773878374650335">"Google taula"</string>
     <string name="gform_file_type" msgid="4803176103746107611">"Google inprimakia"</string>
     <string name="gmap_file_type" msgid="6684180781808007016">"Google mapa"</string>
-    <string name="gsite_file_type" msgid="3742812051249149526">"Google webgunea"</string>
+    <string name="gsite_file_type" msgid="3742812051249149526">"Google-ren webgunea"</string>
     <string name="directory_type" msgid="2702987727566226354">"Karpeta"</string>
 </resources>
diff --git a/res/values-uz/inspector_strings.xml b/res/values-uz/inspector_strings.xml
index 1a791f3e..680c6458 100644
--- a/res/values-uz/inspector_strings.xml
+++ b/res/values-uz/inspector_strings.xml
@@ -16,7 +16,7 @@
 
 <resources xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="inspector_title" msgid="1924760928091740238">"Ma’lumot"</string>
+    <string name="inspector_title" msgid="1924760928091740238">"Axborot"</string>
     <string name="inspector_load_error" msgid="7522190243413249291">"Fayl haqida ma’lumotni yuklab bo‘lmadi"</string>
     <string name="inspector_debug_section" msgid="2576052661505700421">"Xatoliklarni tuzatish (faqat dasturchilar uchun)"</string>
     <string name="inspector_debug_metadata_section" msgid="5875140675600744846">"RAW fayli meta-axboroti: <xliff:g id="METADATATYPE">%1$s</xliff:g>"</string>
diff --git a/res/values-uz/strings.xml b/res/values-uz/strings.xml
index 3cfa1d25..47b367b3 100644
--- a/res/values-uz/strings.xml
+++ b/res/values-uz/strings.xml
@@ -41,7 +41,7 @@
     <string name="menu_compress" msgid="37539111904724188">"Arxivlash"</string>
     <string name="menu_extract" msgid="8171946945982532262">"Arxivdan chiqarish"</string>
     <string name="menu_rename" msgid="1883113442688817554">"Qayta nomlash"</string>
-    <string name="menu_inspect" msgid="7279855349299446224">"Ma’lumot olish"</string>
+    <string name="menu_inspect" msgid="7279855349299446224">"Axborot olish"</string>
     <string name="menu_view_in_owner" msgid="7228948660557554770">"<xliff:g id="SOURCE">%1$s</xliff:g> orqali ochish"</string>
     <string name="menu_new_window" msgid="2947837751796109126">"Yangi oyna"</string>
     <string name="menu_cut_to_clipboard" msgid="2878752142015026229">"Kesish"</string>
-- 
2.17.1

